src/lib/chips/chip-list.ts

```diff
@@ -45,6 +45,7 @@ import {coerceBooleanProperty} from '@angular/cdk/coercion';
   host: {
     '[attr.tabindex]': '_tabIndex',
     'role': 'listbox',
+    'aria-orientation': 'horizontal',
```

Comment:
Stacked chips are vertical.  
I'm okay with the change. But I think we need to update the docs to let developers set correct `aria-orientation` if they use stacked chip list.